shopfloor_reception_packaging_dimension_mobile/static/src/scenario/reception_packaging_dimension.js

```diff
@@ -0,0 +1,144 @@
+                    label="Barcode"
+                    placeholder="Packaging Barcode"
+                    v-model="state.data.packaging.barcode"
+                ></v-text-field>
+            </v-row>
+            <v-row>
+                <v-text-field
+                    label="Quantiy"
+                    type="number"
+                    placeholder="Packaging Quantity"
+                    v-model="state.data.packaging.qty"
+                ></v-text-field>
+            </v-row>
+            <v-row>
+                <v-text-field
+                    label="Length"
+                    type="number"
+                    :suffix="state.data.packaging.length_uom"
+                    placeholder="Packaging Length"
+                    v-model="state.data.packaging.length"
+                ></v-text-field>
+            </v-row>
+            <v-row>
+                <v-text-field
+                    label="Width"
+                    type="number"
+                    :suffix="state.data.packaging.length_uom"
+                    placeholder="Packaging Width"
+                    v-model="state.data.packaging.width"
+                ></v-text-field>
+            </v-row>
+            <v-row>
+                <v-text-field
+                    label="Height"
+                    type="number"
+                    :suffix="state.data.packaging.length_uom"
+                    placeholder="Packaging Height"
+                    v-model="state.data.packaging.height"
+                ></v-text-field>
+            </v-row>
+            <v-row>
+                <v-text-field
+                    label="Max Weight"
+                    type="number"
+                    :suffix="state.data.packaging.weight_uom"
+                    placeholder="Packaging Max Weight"
+                    v-model="state.data.packaging.max_weight"
+                ></v-text-field>
+            </v-row>
+            <!-- extend -->
+       </v-container>
+    </v-form>
+
+    <div class="button-list button-vertical-list full">
+        <v-row align="center">
+            <v-col class="text-center" cols="12">
+                <btn-action action="todo" @click="state.on_done">Done</btn-action>
+            </v-col>
+        </v-row>
+        <v-row align="center">
+            <v-col class="text-center" cols="12">
+
+                <btn-action color="default" @click="state.on_skip">Skip</btn-action>
+
+            </v-col>
+        </v-row>
+    </div>
+</div>
+
+` +
+    template.substring(pos);
+
+// Extend the reception scenario with :
+//   - the new patched template
+//   - the js code for the new state
+const ReceptionPackageDimension = process_registry.extend("reception", {
+    template: new_template,
+    "methods.get_packaging_measurements": function () {
+        return ["length", "width", "height", "max_weight", "qty", "barcode"];
+    },
+    "methods._get_states": function () {
+        let states = reception_states.bind(this)();
```

Comment:
```suggestion
        let states = _get_states.bind(this)();
```
Otherwise if an other module tries to overwrite the states, it will not work, because always the one from the reception_states are used + the new state which is defined here.

Reply:
changed